CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.0.35-dev0
+## 0.0.35
```

Comment:
this one only can have dev version, and we bump this to non dev when we are ready to publish 0.0.35

Reply:
Oh, but then `check-version` would be out of sync right?

Reply:
nope, they won't check changlog, check-version check if the version is in sync between the yaml file and the version in general api